components/tools/OmeroWeb/omeroweb/webclient/tree.py

```diff
@@ -1787,7 +1787,7 @@ def marshal_annotations(conn, project_ids=None, dataset_ids=None,
             join fetch ch.details.creationEvent
             join fetch ch.details.owner
             left outer join fetch ch.file as file
-            where %s
+            where %s order by ch.ns
```

Comment:
order of maps is hardcoded by namespace, I found that the most suitable and guarantee the same order across all the studies. Not sure the best solution